meinberlin/assets/scss/components_user_facing/_a4-comments.scss

```diff
@@ -1,94 +1,233 @@
+        margin-right: 12px;
+    }
+
+    > li {
+        margin-bottom: 0;
+
+        &:after {
+            @include pseudo-border;
+        }
+
+        &:first-child:before {
+            @include pseudo-border;
+        }
+    }
+
+    @media screen, (max-width: $breakpoint) {
+        width: calc(100% + 24px);
+        margin-left: -12px;
+        margin-right: -12px;
+    }  
+}
+
+.a4-comments__box {
+    padding-top: $em-spacer;
+
+    .row {
+        position: relative;
+    }
+
+    @media screen, (max-width: $breakpoint) {
+        padding-left: 12px;
+        padding-right: 12px;
+    }  
+}
+
 .a4-comments__commentbox__title-comments {
-  display: none;
+    display: none;
 }
 
 .a4-comments__commentbox__subtitle {
-  margin-top: 2 * $spacer;
-  margin-bottom: 0.5 * $spacer;
+    margin-top: 2 * $spacer;
+    margin-bottom: 0.5 * $spacer;
 }
 
 .a4-comments__anchor {
-  display: block;
-  position: relative;
-  top: -100px;
-  visibility: hidden;
-  height: 1px;
+    display: block;
+    position: relative;
+    top: -100px;
+    visibility: hidden;
+    height: 1px;
 }
 
 .a4-comments__submit-input {
-  @extend .button;
-  @extend .button--fullwidth-palm;
+    @extend .button;
+    @extend .button--fullwidth-palm;
 }
 
 .a4-comments__cancel-edit-input {
-  @extend .button;
-  @extend .button--light;
+    @extend .button;
+    @extend .button--light;
 
-  @media screen and (max-width: $breakpoint-palm) {
-    margin-bottom: 0 !important;
-  }
+    @media screen and (max-width: $breakpoint-palm) {
+        margin-bottom: 0 !important;
+    }
 }
 
 .a4-comments__comment-form__heading-comments-allowed {
-  @extend h2;
-  margin-bottom: 0.75 * $spacer;
+    @extend h2;
+    margin-bottom: 0.75 * $spacer;
 }
 
 .a4-comments__comment-form__actions {
-  @extend .form-actions;
-  margin-top: 0.25 * $spacer;
+    @extend .form-actions;
+    margin-top: 0.25 * $spacer;
 }
 
 .a4-comments__comment-form__actions__left {
-  @extend .form-actions__left;
+    @extend .form-actions__left;
 
-  @media screen and (max-width: $breakpoint-palm) {
-    margin-bottom: 0 !important;
-  }
+    @media screen and (max-width: $breakpoint-palm) {
+        margin-bottom: 0 !important;
+    }
 }
 
 .a4-comments__comment-form__actions__right {
-  @extend .form-actions__right;
+    @extend .form-actions__right;
 
-  @media screen and (max-width: $breakpoint-palm) {
-    margin-top: 0 !important;
-    margin-bottom: 0 !important;
-  }
+    @media screen and (max-width: $breakpoint-palm) {
+        margin-top: 0 !important;
+        margin-bottom: 0 !important;
+    }
 }
 
 .a4-comments__comment-form__form {
-  @extend .panel--heavy;
-  margin-bottom: $spacer;
+    @extend .panel--heavy;
+    margin-bottom: $spacer;
 
-  .form-group {
-    margin-bottom: 0;
-  }
+    .form-group {
+        margin-bottom: 0;
+    }
 
-  .a4-comments__char-count {
-    @extend .formgroup__help;
-    color: var(--color-black);
-    margin-top: 0.25 * $spacer;
+    &__char-count {
```

Comment:
this doesn't seem to work, the char count doesn't get any of the styling

Reply:
ah ok, in firefox I don't get a horizontal scrollbar so they just disappear to the right (share button)

Reply:
@hom3mad3 sorry, it actually works, I was just too stupid to trigger it!